judge/runpipe.cc

```diff
@@ -641,22 +641,22 @@ struct state_t {
         tie(read_end, write_end) = make_pipe();
         logmsg(LOG_DEBUG, "setting up pipe #%ld (fd %d) -> proxy (fd %d)", i,
                write_end, read_end);
-        process.stdout = write_end;
+        process.pstdout = write_end;
```

Comment:
What does `p` stand for? I think this needs a clearer variable name. I don't know the context well enough to choose the best one, though.

Reply:
I agree. There is the need to rename this variable, but I don't like `pstdin`/`pstdout` either. Here some suggestions (in my decreasing order of preference):

- `standard_input`
- `stdin_fd`
- `input`
- `stdin_`

Reply:
I'm fine with either `standard_{in,out}put` or `std{in,out}_fd`, @eldering do you have a preference? In the end it should be clear for the person doing the debugging which will probably not be me.

If no response by wednesday I'll pick `stdin_fd` and merge (as its the shortest and its trivial to rename later and I need this for my next PR).